src/platform/packages/shared/kbn-rrule/rrule.ts

```diff
@@ -134,6 +138,16 @@ export class RRule {
       return dates;
     }
   }
+
+  static isValid(options: ConstructorOptions): boolean {
+    try {
+      validateOptions(options);
+
+      return true;
```

Comment:
I'd prefer to find a way to surface the error message that `validateOptions` produces instead of just turning it into a boolean. Not a blocker for this PR because I see we're just using it to display a notification badge. If there's a way to surface the error message without a huge rewrite I'd say give it a shot, otherwise feel free to merge.

Reply:
Hey Zacq! In my use case, I do not want to surface the message but to know which rules have an invalid snooze schedule (very rare after the guardrails we put) and show an "Invalid Snooze" notification badge which the user can click and remove. Wdyt?

Reply:
Sounds all right, we've got the bones in place for surfacing the error message in future use cases